docs/src/03-tutorials/01-tutorials.md

```diff
@@ -0,0 +1,11 @@
+# GraphFrames Tutorials
+
+This section contains step-by-step tutorials to help you learn and master specific features of GraphFrames.
+
+Currently available tutorials:
+
+* [Motif Finding Tutorial](/03-tutorials/02-motif-tutorial.md): Learn how to perform pattern recognition in graphs using network motif finding. This
+  tutorial demonstrates the technique using the `stackexchange.com` knowledge graph, showing you how to identify and
+  analyze recurring patterns and substructures within the graph data.
+
+More tutorials will be added in future releases to cover additional GraphFrames features and use cases.
```

Comment:
I thought I had https://docs.databricks.com/aws/en/integrations/graphframes/ in there as How to use GraphFrames on Databricks?

Reply:
Added